packages/sysutils/systemd/scripts/module-overlays-setup

```diff
@@ -0,0 +1,77 @@
+#!/bin/sh
+################################################################################
+#      This file is part of LibreELEC - http://www.libreelec.tv
+#      Copyright (C) 2017 Team LibreELEC
+#
+#  LibreELEC is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  LibreELEC is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with LibreELEC.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+
+SYSTEM_OVERLAYS_DIR=/usr/lib/module-overlays
+OVERLAY_CONFIG_DIR=/storage/.cache/module-overlays
+KVER=$(uname -r)
+MODULES_DIR="/var/lib/modules/${KVER}"
+
+mkdir -p "${MODULES_DIR}"
+mkdir -p "${OVERLAY_CONFIG_DIR}"
+
+log() {
+  echo "module-overlays-setup: $@" > /dev/kmsg
+}
+
+apply_overlay() {
+ case "$1" in
+    /*)
+      overlay_dir="${1}/lib/modules/${KVER}"
+      ;;
+    *)
+      overlay_dir="${SYSTEM_OVERLAYS_DIR}/${1}/lib/modules/${KVER}"
+      ;;
+  esac
+
+  if [ ! -d "${overlay_dir}" ] ; then
+    return 1
+  fi
+
+  cp -rfs "${overlay_dir}"/* "${MODULES_DIR}"
+}
+
+# setup system base modules
+
+log "setup base modules"
+apply_overlay base
+
+# apply user-configured module overlays
+
+if [ -d "${OVERLAY_CONFIG_DIR}" ] ; then
+  log "adding overlays from ${OVERLAY_CONFIG_DIR}"
+  got_overlays="no"
+  for conf in "${OVERLAY_CONFIG_DIR}/"*.conf ; do
```

Comment:
Good catch! I missed that during testing. I'll add an explicit check if the glob went through without match.

BTW: using find won't work, as that won't return the matches sorted by name/number. Globs OTOH will do that